<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vehicle Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(227, 227, 227)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="file_selector_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 170, 255);
color:rgb(255, 255, 255);
font:15px;</string>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Selected File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_name_container">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Signals</string>
     </property>
    </widget>
    <widget class="QComboBox" name="signal_list_dropdown">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>481</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:rgb(255,255,255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:bold;
font:18px;</string>
     </property>
     <property name="text">
      <string>Signal Information</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:bold;
font:14px</string>
     </property>
     <property name="text">
      <string>Signal Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:14px</string>
     </property>
     <property name="text">
      <string>Signal Type:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="signal_info_name">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="signal_info_type">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="send_signal_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:12px;
font-weight:bold;
background-color:rgb(85, 170, 255);
color:white</string>
     </property>
     <property name="text">
      <string>Send Signal</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Background-color:rgb(255,255,255);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QRadioButton" name="signal_true_radio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>62</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:12px;
font-weight:bold</string>
       </property>
       <property name="text">
        <string>True</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="signal_false_radio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>62</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:12px;
font-weight:bold</string>
       </property>
       <property name="text">
        <string>False</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QSpinBox" name="signal_value_int">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:12px;
background-color:rgb(85, 170, 255);
font-weight:bold;
color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QComboBox" name="signal_value_enum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:12px;
background-color:rgb(85, 170, 255);
font-weight:bold;
color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QDoubleSpinBox" name="signal_value_double">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font:12px;
background-color:rgb(85, 170, 255);
font-weight:bold;
color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>761</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Background-color:rgb(255,255,255);
border:0px</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font:12px;
font-weight:bold;
</string>
      </property>
      <property name="text">
       <string>[SIGNAL INFO]</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVehicle_signal_simulator">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBrowse_file"/>
   </widget>
   <addaction name="menuVehicle_signal_simulator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBrowse_file">
   <property name="text">
    <string>Browse file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
